ui/lib/pki/addon/components/pki-generate-toggle-groups.hbs

```diff
@@ -39,7 +39,6 @@
           {{else if (eq group "Additional subject fields")}}
             These fields provide more information about the client to which the certificate belongs.
           {{/if}}
-          Add one item per row.
```

Comment:
Instead of in this description for the toggle group, this now renders for each input. 
<img width="1000" alt="Screenshot 2023-04-13 at 4 30 54 PM" src="https://user-images.githubusercontent.com/68122737/231904545-b938bf51-8530-4132-b0dc-5fc822998696.png">